siriuspy/siriuspy/cycle/main.py

```diff
@@ -744,24 +744,25 @@ def check_pwrsupplies_slowref(self, psnames, timeout=TIMEOUT_CHECK):
 
         self._update_log('Checking power supplies opmode...')
         self._checks_result = {psn: False for psn in psnames}
+        msg = 'Successfully checked opmode preparation for {}/' + \
+            str(len(psnames))
         time = _time.time()
         while _time.time() - time < timeout:
-            for idx, psname in enumerate(psnames):
+            for psname in psnames:
                 if self._checks_result[psname]:
                     continue
                 cycler = self._get_cycler(psname)
                 if cycler.check_opmode_slowref(0.05):
                     self._checks_result[psname] = True
                     checked = sum(self._checks_result.values())
-                    if checked % 5 == 0 or idx == len(psnames)-1:
-                        self._update_log(
-                            'Successfully checked opmode preparation for ' +
-                            '{0}/{1}'.format(str(checked), str(len(psnames))))
+                    if checked % 5 == 0:
```

Comment:
I think it is cleaner if you do `if not checked % 5`

Reply:
done!